dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -529,10 +529,16 @@ class Timeseries {
 
     // Final statistics.
     final double cleanAverage = _computeAverage(name, cleanValues);
-    final double outlierAverage = _computeAverage(name, outliers);
     final double standardDeviation = _computeStandardDeviationForPopulation(name, cleanValues);
     final double noise = cleanAverage > 0.0 ? standardDeviation / cleanAverage : 0.0;
 
+    // Compute outlier average. If there are no outliers the outlier average is
+    // the same as clean value average. In other words, in a perfect benchmark
+    // with no noise the difference between average and outlier average is zero,
+    // which the best possible outcome. Noise produces a positive difference
+    // between the two.
```

Comment:
I'm not sure how this will be the outlier average will be interpreted in the dashboard. Generally, in the dashboard, the lower the better. So I think the perfect case should be set to zero. Open to talk about it.

Reply:
We might be talking about two different perfections. One is "a perfect Web engine is so fast that it spends zero amount of time to compute everything". The other is "a perfect benchmark has no outliers" (even if the engine is not perfect and does require time compute stuff). This one's about the latter, not the former. When there are no outliers, it means the benchmark is not noisy, and therefore there's no difference between the two averages.

Reply:
Agreed that the two perfections are different.

So how am I supposed to look at the outliers card in the benchmarks dashboard? It doesn't mean anything when it's alone. It has to be seen alongside the clean average card. This makes me think, maybe reporting the outlier average wasn't the right choice. What if we report The difference between outlier average and clean average? That way I can look at the outlier card alone and make sense of it (the higher it is, the worse the outliers are) and I don't have to compare it with anything else.

Reply:
Discussed offline and decided a good metric would the ratio `outlierAverage:cleanAverage`. But that can be done as a separate PR, so this comment is non-blocking.

Reply:
Done: https://github.com/flutter/flutter/pull/55126/commits/19c691bc3171a2d12f271836de104cf2ef204466